frontend/src/pages/TermPlanner/DraggableCourse/styles.js

```diff
@@ -56,4 +58,13 @@ const CourseLabel = styled.div`
   margin: 0px;
 `;
 
-export default { CourseWrapper, CourseLabel };
+const Text = styled(Typography)`
```

Comment:
Think you can remove this due to the mentioned use of `<Text classname="text" />` instead?